### ISSUE TITLE
Parse error for inline comments

### ISSUE THREAD

**Issue by aliaksandrb** [sass/node-sass#826]
Hi guys!

Here is a bug (I believe) I found recently:
Imagine we have `.scss` file with such content:
```scss
button,
html input[type="button"], /* 1 */
input[type="reset"],
input[type="submit"] {
  -webkit-appearance: button; /* 2 */
  cursor: pointer; /* 3 */
}
```

If we try to build it with a script:
```javascript
var sass = require('node-sass');

var result = sass.renderSync({
  file: 'file.scss',
  outputStyle: 'compressed',
  outFile: 'result.css',
  sourceMap: true
});

console.log(result.css);
console.log(result.map);
console.log(result.stats);
```

We get an error:
```javascript
/node_modules/node-sass/lib/index.js:366
  throw util._extend(new Error(), JSON.parse(result.error));
             ^
Error: invalid top-level expression
    at Object.module.exports.renderSync (/node_modules/node-sass/lib/index.js:366:22)
    at Object.<anonymous> (/test.js:3:19)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
    at startup (node.js:129:16)
    at node.js:814:3
```

But if we remove the inline comment at line with selector:
```scss
button,
html input[type="button"],
input[type="reset"],
input[type="submit"] {
  -webkit-appearance: button; /* 2 */
  cursor: pointer; /* 3 */
}
```
Than the build successfull:

```javascript
<Buffer 62 75 74 74 6f 6e 2c 68 74 6d 6c 20 69 6e 70 75 74 5b 74 79 70 65 3d 22 62 75 74 74 6f 6e 22 5d 2c 69 6e 70 75 74 5b 74 79 70 65 3d 22 72 65 73 65 74 ... >
<Buffer 7b 0a 09 22 76 65 72 73 69 6f 6e 22 3a 20 33 2c 0a 09 22 66 69 6c 65 22 3a 20 22 73 61 73 68 61 2e 63 73 73 22 2c 0a 09 22 73 6f 75 72 63 65 73 22 3a ... >
{ entry: 'file.scss',
  start: 1427819797754,
  includedFiles: [ 'file.scss' ],
  end: 1427819797756,
  duration: 2 }
```

The interesting thing that it reproducible with `node-sass v3`
└── node-sass@3.0.0-beta.4 extraneous (git://github.com/sass/node-sass#5d5dff405493625edee0352b061729db0ce03c30)

But isn't with `node-sass v2`..

And the main point (because actually I prefer not to use inline comments on lines with selectors) that  source of `.scss` is famous `normalize.css` (scss) https://github.com/necolas/normalize.css/blob/master/normalize.css#L291 uses in many ui-libraries.

Does it make sense?

Thanks!

**Comment by mgreter** [comment]
This is already fixed in upstream and included in libsass 3.2.0-beta.4 (https://github.com/sass/libsass/pull/1008).

**Comment by aliaksandrb** [comment]
@mgreter great news! You are rock :metal:.
